app/locales/sk/locationTracking.json

```diff
@@ -0,0 +1,13 @@
+{
+    "start_logging":"ZAČAŤ ZAZNAMENÁVAŤ",
+    "stop_logging":"PRESTAŤ ZAZNAMENÁVAŤ",
+    "logging_message":"Práve sa súkromne zaznamenáva tvoja poloha každých päť minút. Informácie o tvojej polohe NEOPUSTIA tvoj telefón.",
+    "not_logging_message":"POZNÁMKA: Po kliknutí na toto tlačidlo sa bude môcť zobraziť výzva na poskytnutie Private Kit prístup k tvojej polohe.",
```

Comment:
prístupu, not prístup

Reply:
Consider changing "k tvojej" to "k vašej" for a more formal tone of voice